runtime/src/main/java/com/datastax/oss/quarkus/runtime/driver/QuarkusNettyOptions.java

```diff
@@ -0,0 +1,164 @@
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.driver;
+
+import com.datastax.oss.driver.api.core.config.DefaultDriverOption;
+import com.datastax.oss.driver.api.core.config.DriverExecutionProfile;
+import com.datastax.oss.driver.internal.core.context.NettyOptions;
+import com.datastax.oss.driver.internal.core.util.concurrent.BlockingOperation;
+import com.datastax.oss.driver.shaded.guava.common.util.concurrent.ThreadFactoryBuilder;
+import io.netty.bootstrap.Bootstrap;
+import io.netty.buffer.ByteBufAllocator;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.FixedRecvByteBufAllocator;
+import io.netty.channel.socket.nio.NioSocketChannel;
+import io.netty.util.HashedWheelTimer;
+import io.netty.util.Timer;
+import io.netty.util.concurrent.DefaultPromise;
+import io.netty.util.concurrent.EventExecutorGroup;
+import io.netty.util.concurrent.Future;
+import io.netty.util.concurrent.GlobalEventExecutor;
+import io.netty.util.internal.PlatformDependent;
+import java.time.Duration;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.TimeUnit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class QuarkusNettyOptions implements NettyOptions {
+  private static final Logger LOG = LoggerFactory.getLogger(QuarkusNettyOptions.class);
+
+  private final DriverExecutionProfile config;
+  private final EventLoopGroup ioEventLoopGroup;
+  private final EventLoopGroup adminEventLoopGroup;
+  private final Timer timer;
+
+  public QuarkusNettyOptions(
+      QuarkusDriverContext context,
+      EventLoopGroup ioEventLoopGroup,
+      EventLoopGroup adminEventLoopGroup) {
+    this.config = context.getConfig().getDefaultProfile();
+    boolean daemon = config.getBoolean(DefaultDriverOption.NETTY_DAEMON);
+    // the NETTY_IO_SHUTDOWN_QUIET_PERIOD, NETTY_IO_SHUTDOWN_TIMEOUT, NETTY_IO_SHUTDOWN_UNIT,
+    // NETTY_ADMIN_SHUTDOWN_QUIET_PERIOD,
+    // NETTY_ADMIN_SHUTDOWN_TIMEOUT, NETTY_ADMIN_SHUTDOWN_UNIT
+    // are ignored when using quarkus netty even loops
+
+    ThreadFactory safeFactory = new BlockingOperation.SafeThreadFactory();
+
+    this.ioEventLoopGroup = ioEventLoopGroup;
+    this.adminEventLoopGroup = adminEventLoopGroup;
+
+    // setup the Timer
+    ThreadFactory timerThreadFactory =
+        new ThreadFactoryBuilder()
+            .setThreadFactory(safeFactory)
+            .setNameFormat(context.getSessionName() + "-timer-%d")
+            .setDaemon(daemon)
+            .build();
```

Comment:
```suggestion
    this.ioEventLoopGroup = ioEventLoopGroup;
    this.adminEventLoopGroup = adminEventLoopGroup;

    // setup the Timer
    ThreadFactory timerThreadFactory =
        new ThreadFactoryBuilder()
            .setThreadFactory(new BlockingOperation.SafeThreadFactory())
            .setNameFormat(context.getSessionName() + "-timer-%d")
            .setDaemon(config.getBoolean(DefaultDriverOption.NETTY_DAEMON))
            .build();
```

Reply:
This is copy-paste from the original `DefaultNettyOptions` so I would like to keep it in sync if possible